README.md

```diff
@@ -30,24 +30,35 @@ Complete configuration for the different [node.js versions](https://github.com/n
 # https://github.com/nodejs/LTS
 sudo: false
 language: node_js
+os:
+ - linux
+before_install:
+  - 'case "${TRAVIS_NODE_VERSION}" in 0.*) export NPM_CONFIG_STRICT_SSL=false ;; esac'
```

Comment:
this avoids SSL errors when connecting to the npm registry in the latest npm versions that 0.x versions of node support